oauth2client/contrib/appengine.py

```diff
@@ -89,11 +89,11 @@ def _safe_html(s):
     return cgi.escape(s, quote=1).replace("'", '&#39;')
 
 
-class InvalidClientSecretsError(Exception):
-    """The client_secrets.json file is malformed or missing required fields."""
+class Error(Exception):
+    """Base error for this module."""
```

Comment:
["Modules or packages should define their own domain-specific base exception class, which should inherit from the built-in `Exception` class. The base exception for a module should be called `Error`."](https://google.github.io/styleguide/pyguide.html?showone=Exceptions#Exceptions) is a terrible practice and I've been trying for years to get it removed from the style guide. Thank you for not following it. :-)

I apologize that there are parts of the codebase that define such nonsense per-module base exception classes.